app/auth/models.py

```diff
@@ -152,3 +153,42 @@ def create_user(self, email, full_name, user_name, password):
             return response.json()
         else:
             return None
+
+
+class FileData(object):
```

Comment:
Why introduce this new model object?

Did any functionality like this exist before? If so, where and how has that been refactored into this code?

Again a note in the commit message about this would help explain how these changes fit together.